shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -95,7 +112,11 @@ public void startShimmerAnimation() {
             getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
                 @Override
                 public void onGlobalLayout() {
-                    ShimmerLayout.this.getViewTreeObserver().removeOnGlobalLayoutListener(this);
+                    if (Build.VERSION.SDK_INT > 15) {
+                        ShimmerLayout.this.getViewTreeObserver().removeOnGlobalLayoutListener(this);
```

Comment:
It would be nice to create a new method for listener removing like the getColor method.